src/docs/development/ui/navigation/deep-linking.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deep linking
+description: Navigate to routes when the app receives a new URL
+---
+
+{{site.alert.note}}
+  This feature is only available on the dev or master channel. See [Switching
+  Flutter channels][switching-channels] for more information.
+{{site.alert.end}}
+
+Flutter supports deep linking on iOS, Android and web browsers in the dev
+channel. Users opening a URL will display a specific screen in your app. By
+following these steps, your can launch and display routes via named routes
+(either with the [`routes`][routes] parameter or
+[`onGenerateRoute`][onGenerateRoute]), or by using the [Router][] widget.
+
+{{site.alert.secondary}}
+  [Navigator 2.0][] is now called [Router][], which allows you to declaratively
+  set the routes are currently shown based on your app state. This API is
+  opt-in.
```

Comment:
The "secondary" style of note needs bolded title text, which is why we use this type of note.

I know that before I said don't have identical words in the second set of square brackets, but to stick with your style of using the second pair of brackets to avoid code fonting, I changed this text to use that pattern.

```suggestion
  **Version note:** [Navigator 2.0][] is now called [`Router`][Router], which allows
  you to declaratively set the displayed routes based on the app's current state.
  This API is opt-in.
```